src/shelf.js

```diff
@@ -49,7 +49,13 @@ const formatUseCaseDoc = (usecase) => {
 }
 
 function renderShelfHTML(usecases) {
+
   const shelfData = generateShelfData(usecases)
+
+  if (process.env.HERBS_SHELF_RENDER === "false") {
```

Comment:
why not use `if (process.env.HERBS_SHELF_RENDER)` ? 
so in my project I can use HERBS_SHELF_RENDER=true | HERBS_SHELF_RENDER=1
if I want to set this env as false, just remove it like others envs